requirements/strategies.txt

```diff
@@ -0,0 +1,3 @@
+fairscale>=0.4.5
+deepspeed<0.6.0
```

Comment:
```suggestion
deepspeed!=0.6.0,!=0.6.1,!=0.6.2
```

Reply:
or
```suggestion
deepspeed<0.6.0,>=0.6.3
```
(not sure if this is valid though.)

Reply:
let's keep this simple as this is the actual state on master and do this adjustments in separate PR :rabbit:

Reply:
@akihironitta @Borda Let's create a Github issue for this so that we don't miss it?